source/assets/stylesheets/sections/_railsgirls.scss

```diff
@@ -0,0 +1,18 @@
+.rectangle {
+  width: 250px;
+  height: 60px;
+  background: $brand-primary;
+  margin: 0 auto;
+  line-height: 60px;
+}
+
+.social-links {
+  width: 200px;
+  margin: 0 auto;
+}
+
+.imgsize2 {
```

Comment:
This is not a very intention revealing name. To adjust the sizes the images you can use the boostrap grid classes. https://getbootstrap.com/css/#grid